contracts/base/LoansBase.sol

```diff
@@ -280,21 +251,16 @@ contract LoansBase is Base {
         whenLendingPoolNotPaused(address(lendingPool))
         nonReentrant()
     {
-        // TODO Validate allowance amount.
         // calculate the amount of collateral the loan needs in tokens
-        uint256 collateralNeededToken = _getCollateralNeededInTokens(
-            _getTotalOwed(loanID),
-            loans[loanID].loanTerms.collateralRatio
+        (uint256 loanCollateral, , uint256 collateralNeededWei, ) = _getCollateralInfo(
```

Comment:
I think we need to stop using the word 'wei' in this contract when it's not necessarily ETH.

Reply:
I will change it to "collateralNeededInCollTokens".